.github/workflows/IJ-latest.yml

```diff
@@ -23,7 +23,9 @@ jobs:
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
       - name: Build with Gradle
-        run: ./gradlew build --continue -PideaVersion=LATEST-EAP-SNAPSHOT
+        run: |
+          LATEST-EAP-SNAPSHOT=$(./gradlew printProductsReleases | grep 'IC-' | head -n 1 | cut -d'-' -f2)
```

Comment:
looks like "LATEST-EAP-SNAPSHOT" doesnt exist any more since you query and parse?